pkg/api/v1beta3/types.go

```diff
@@ -200,26 +200,26 @@ type VolumeSource struct {
 	// to see the host machine. Most containers will NOT need this.
```

Comment:
Why is ObjectMeta not omitempty? All the fields in ObjectMeta are omitempty.
Not making it omitempty will just force users to include "metadata:{}" in their configs.